pkg/plugins/external/external_test.go

```diff
@@ -679,6 +679,54 @@ var _ = Describe("Run external plugin using Scaffold", func() {
 			checkMetadata()
 		})
 	})
+
+	Context("Helper functions for Sending request to external plugin and parsing response", func() {
+		It("getUniverseMap should return path to content mapping of all files in Filesystem", func() {
+			fs := machinery.Filesystem{
+				FS: afero.NewMemMapFs(),
+			}
+
+			files := []struct {
+				path    string
+				name    string
+				content string
+			}{
+				{
+					path:    "./",
+					name:    "file",
+					content: "level 0 file",
+				},
+				{
+					path:    "dir/",
+					name:    "file",
+					content: "level 1 file",
+				},
+				{
+					path:    "dir/subdir",
+					name:    "file",
+					content: "level 2 file",
+				},
+			}
+
+			// create files in Filesystem
+			for _, file := range files {
+				fs.FS.MkdirAll(file.path, 0o700)
```

Comment:
@em-r,

We will need to sort out the lint issues to move forward.
Could you please address them out?

Reply:
Hi @camilamacedo86, the linting issues are now fixed. thank you!